test/Fail/Issue3606.err

```diff
@@ -1,5 +1,5 @@
 Failed to solve the following constraints:
-  fst ?0 = fst x : A (blocked on _5)
+  ?0 .fst = fst x : A (blocked on _5)
```

Comment:
Why is the first `fst` here converted to postfix syntax but the second one not?

Reply:
The user wrote a prefix projection in `fst x`:
```agda
test : (x : ΣAP) → P (fst x)
test x = snd {!!}
```
These are not touched by the `--postfix-projection` option.
If the user wants a consistent experience, they should use postfix projections now always or switch to `--no-postfix-projections`.

Reply:
Ok, I see.